pluma/pluma-pango.c

```diff
@@ -103,9 +103,14 @@ pluma_pango_font_description_to_css (const PangoFontDescription *desc)
   set = pango_font_description_get_set_fields (desc);
   if (set & PANGO_FONT_MASK_FAMILY)
     {
-      g_string_append (s, "font-family: ");
-      g_string_append (s, pango_font_description_get_family (desc));
-      g_string_append (s, "; ");
+      const char* font_familay = pango_font_description_get_family (desc);
+      if (font_familay[0] != '"') {
```

Comment:
Why add the ``` if (font_familay[0] != '"')```

Reply:
Because `font-family: "Sans"` is valid. So if we does  not check if there is a quote, we will make new CSS code like this: `font_familay: ""Sans""`

Reply:
How can I generate font ```family: "Sans"```.I cannot set the font to ```"Sans"```

Reply:
Sorry, I have made a mistake. I have tested that the font family name from Pango is without quotes.
So, I have reverted the code.

Reply:
1. In pluma, the font name is without quote by default.

2.  We could use the font name without quote to call pango_font_description_from_string function in order to get a PangoFontDescription object.  Therefore, if we use the previous `PangoFontDescription `object to call `pango_font_description_get_family `function, we will get a font family name without quote.

3. We could use a font name with quote to call pango_font_description_from_string function in order to get a PangoFontDescription object.  Therefore, if we use the previous `PangoFontDescription `object to call `pango_font_description_get_family `function, we will get a font family name with quote.

4.  So, we could modify `pluma_pango_font_description_to_css `function in order to have less effect on other codes.